_posts/2018-10-02-auto-tune-all.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: post
+title: Automatic Kernel Optimization for Deep Learning on All Hardware Platforms
+date: 2018-10-02
+author: Lianmin Zheng, Eddie Yan
+tags:
+  - tvm
+---
+
+How to optimize the performance of deep neural network on a diverse range of hardware platforms is still a hard
+problem for AI developers. In terms of system support, we are facing a many-to-many problem here:
+deploying trained models from multiple frontends (e.g. Tensorflow, ONNX, MXNet) to multiple
+hardware platforms (e.g. CPU, GPU, Accelerators). On the most performance critical part of
+this problem is how to get high performance kernel implementation for growing model
+architectures and hardware platforms.
+
+To address this challenge, TVM takes a full stack compiler approach. Combining code generator and auto-tuner in TVM,
```

Comment:
TVM combines code generation and auto-tuning to generate kernels... , obtaining state-of-the-art inference performance including...